src/util/misc/projectStroke/StrokeLineJoinProjections.ts

```diff
@@ -97,7 +97,18 @@ export class StrokeLineJoinProjections extends StrokeProjectionsBase {
    * @see https://github.com/fabricjs/fabric.js/pull/8344#2-2-bevel
    */
   projectBevel() {
-    return [this.B, this.C].map((to) => this.projectOrthogonally(this.A, to));
+    const projections: Point[] = [];
+    [this.B, this.C].map((to) => {
+      projections.push(
+          this.projectOrthogonally(this.A, to)
+      );
+      // if `alpha` equals 0, we need to project for both sides.
+      this.alpha === 0 &&
```

Comment:
> Is this part still needed?

Returning a favor
Since we reduce points it is correct to go with your suggestion and run this logic outside the loop

Reply:
makes sense